include/sddf/i2c/queue.h

```diff
@@ -42,19 +42,16 @@ enum i2c_token {
     /* START: Begin a transfer. Causes master device to capture bus. */
     I2C_TOKEN_START = 0x1,
     /* ADDRESS WRITE: Used to wake up the target device on the bus.
-     * Sets up and following DATA tokens to be writes. */
+     * The byte immediately following this token is an integer (N) length of the succeeding
+       write. Max 256 (1**8). The next N bytes are the payload. */
```

Comment:
If there is a maximum, should the virtualiser be checking it before passing it do the driver?

Reply:
Nevermind, it's a 8-bit integer. Although that does mean that the maximum is 255 not 256.